serving/docker/lmi.Dockerfile

```diff
@@ -106,7 +106,7 @@ RUN pip3 install torch==${torch_version} torchvision==${torch_vision_version} --
     pip3 install ${djl_converter_wheel} --no-deps && \
     pip3 cache purge
 
-RUN pip3 install ${flash_attn_2_wheel} ${lmi_dist_wheel} ${vllm_wheel} pydantic==${pydantic_version} && \
+RUN pip3 install ${flash_attn_2_wheel} ${lmi_dist_wheel} vllm==0.5.0.post1 pydantic==${pydantic_version} && \
```

Comment:
Added this to unblock integration testing. Will remove once the testing is done.